docs/sources/datasources/pyroscope/configure-pyroscope-data-source.md

```diff
@@ -42,16 +42,85 @@ To configure basic settings for the data source, complete the following steps:
-   | `Default`      | The default data source will be pre-selected for new panels.                                                                                                                                                                                                                                                 |
-   | `URL`          | The URL of the Grafana Pyroscope instance, for example, `http://localhost:4100`.                                                                                                                                                                                                                             |
-   | `Basic Auth`   | Enable basic authentication to the data source.                                                                                                                                                                                                                                                              |
-   | `User`         | User name for basic authentication.                                                                                                                                                                                                                                                                          |
-   | `Password`     | Password for basic authentication.                                                                                                                                                                                                                                                                           |
-   | `Minimal step` | Used for queries returning timeseries data. The Pyroscope backend, similar to Prometheus, scrapes profiles at certain intervals. To prevent querying at smaller interval, use Minimal step same or higher than your Pyroscope scrape interval. This prevents returning too many data points to the frontend. |
+## Configuration options
+
+You can configure several options for the Pyroscope data source, including the name, HTTP, authentication, querying, and private data source connect.
+
+If you make any changes, select **Save & test** to preserve those changes.
+
+![Configuration options for the Pyroscope data source](/media/docs/grafana/data-sources/screenshot-pyroscope-data-source-config.png)
+
+### Name and default
+
+**Name**
+: Enter a name to specify the data source in panels, queries, and Explore.
+
+**Default**
+: The default data source is pre-selected for new panels.
+
+### HTTP
+
+The HTTP section is shown in number 1 in the screenshot.
+
+**URL**
+: The URL of the Grafana Pyroscope instance, for example, `https://localhost:4100`.
+
+**Allowed cookies**
+: The Grafana Proxy deletes forwarded cookies. Use this field to specify cookies by name that should be forwarded to the data source.
+
+**Timeout**
+: HTTP request timeout in seconds.
+
+### Auth
```

Comment:
There's a mixture of casing in the description list but I'm gonna assume that matches the UI

Reply:
Yes. It's one of those things we can work on later.

Reply:
Okay, so the data source screen has changed since I started this PR. 
![image](https://github.com/grafana/grafana/assets/104772500/9b0b2ceb-9bfd-4587-83c9-21149ee26516)

I need to update the instructions and the titles.

Reply:
This was the old screen 
![image](https://github.com/grafana/grafana/assets/104772500/aa708e5c-8605-4422-839d-82ceb949ae01)

Reply:
Okay, so the data source was updated and will be doing out with the next release. It is probably useful to merge these changes for 11.1 and then update the data source docs for the next release.